front/src/applications/stdcmV2/views/StdcmViewV2.tsx

```diff
@@ -28,6 +40,18 @@ const StdcmViewV2 = () => {
           pathProperties={pathProperties}
           setPathProperties={setPathProperties}
           setCurrentSimulationInputs={setCurrentSimulationInputs}
+          setStdcmSimulationResults={setStdcmSimulationResults}
+          setSimulationCreationDate={setSimulationCreationDate}
+        />
+      )}
+      {trainScheduleV2Activated && rollingStock && stdcmSimulationResults && (
+        <StdcmResults
+          // TODO: Next step : use currentSimulationInputs instead of stdcmSimulationResults to handle multiples simulations results
+          stdcmData={stdcmSimulationResults}
+          pathProperties={pathProperties}
+          rollingStockData={rollingStock}
+          speedLimitByTag={speedLimitByTag}
+          simulationCreationDate={simulationCreationDate}
```

Comment:
Same, as you can have the creationDate in stdcmData, you can remove this props and later get it with `stdcmResponse.creationDate`
```suggestion
```

Reply:
fix